Makefile.toml

```diff
@@ -222,9 +224,15 @@ dependencies = ["build-x86_64-unknown-linux-gnu-debug"]
 run_task = "run-godot"
 
 [tasks.run-for-macos]
+# dependencies = ["build-aarch64-apple-darwin-debug"]
```

Comment:
Can be removed since both macOS tasks are separated.

Reply:
Done